package_schemas/general_data/2_restricted_schema.json

```diff
@@ -51,6 +51,14 @@
             "private": false,
             "visibility_classes": ["hidden"]
         },
+        {
+            "field_name": "org_to_allow_transfer_to",
+            "label": "Transfer to",
+            "preset": "org_to_allow_transfer_to",
+            "private": false,
+            "visibility_classes": ["hidden"],
+            "display_group": "Inputs to UNAIDS Estimates 2020"
```

Comment:
So this line looks wrong to me... It's package specific, and in the wrong package.